pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
```

Comment:
Spacing seems a bit weird, here's a suggestion:
```nix
{ stdenv, fetchFromGitHub, cmake }:

stdenv.mkDerivation rec {
  pname = "LAStools";
  version = "201003";
```
(Looks fine after that).

Reply:
fixed in a2058a114fcedce1f48c6f8bf5b7967e42dec81b